### PR TITLE
fix(web): add redirect to page after login

### PR DESCRIPTION
**Pull request by jainpawan21**
### What changed? Why was the change needed?
1. added logger in case of hubspot failed to add email to list
2. add `_organizationId: subscriberSession._organizationId,` for widgets analytics event
3. redirect to required page after login

Ref:- [NV-3697](https://linear.app/novu/issue/NV-3697/invite-team-member-nudge)

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/widgets/widgets.controller.ts [hunk 1]
```diff
@@ -476,6 +476,7 @@ export class WidgetsController {
   ): Promise<LogUsageResponseDto> {
     this.analyticsService.track(body.name, subscriberSession._organizationId, {
       environmentId: subscriberSession._environmentId,
+      _organizationId: subscriberSession._organizationId,
```
Comment (jainpawan21) [inline apps/api/src/app/widgets/widgets.controller.ts]: doubt: it should be `_organizationId` or `organization` ?

File: apps/api/src/app/widgets/widgets.controller.ts [hunk 2]
```diff
@@ -476,6 +476,7 @@ export class WidgetsController {
   ): Promise<LogUsageResponseDto> {
     this.analyticsService.track(body.name, subscriberSession._organizationId, {
```
Comment (jainpawan21) [inline apps/api/src/app/widgets/widgets.controller.ts:477]: suggestion : `subscriberSession._organizationId` should be changed with `subscriberSession._userId` as per `track` method params

File: apps/api/src/app/widgets/widgets.controller.ts (hunk 1 above)
Comment (LetItRock) [reply apps/api/src/app/widgets/widgets.controller.ts]: I see that we are not consistent with the pattern, but let's say do `_organization` as I see it's used in payments as well which is pretty new code :D

File: apps/api/src/app/widgets/widgets.controller.ts (hunk 2 above)
Comment (rifont) [reply apps/api/src/app/widgets/widgets.controller.ts:477]: Nice catch

**Review by rifont — APPROVED** [review]

**Pull request merged**